docs/plugins/substitute.rst

```diff
@@ -0,0 +1,27 @@
+Substitute Plugin
+==============
+
+The ``substitute`` plugin lets you easily substitute values in your templates and
```

Comment:
Instead of copying the text from the `rewrite` plugin's documentation, let's refer to (and link to) it and explain how this plugin differs from that.

Reply:
I changed the description